### PR TITLE
Release 2.9.0

### PR DESCRIPTION
**Pull request by brunobat**

### CODE REVIEW

File: .github/project.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 name: SmallRye Config
```
Comment (jasondlee) [inline .github/project.yml:1]: Config?

File: .github/project.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 release:
-  current-version: 2.8.1
-  next-version: 2.8.2-SNAPSHOT
+  current-version: 2.9.0
```
Comment (jasondlee) [inline .github/project.yml:3]: Are we sure we want to bump the minor version here?

**Review by jasondlee — CHANGES_REQUESTED** [review]

File: .github/project.yml (hunk 1 above)
Comment (radcortez) [reply .github/project.yml:1]: I guess it was a copy from there, but it has no effect on the release. Regardless, it should be fixed.

File: .github/project.yml (hunk 2 above)
Comment (rhusar) [reply .github/project.yml:3]: 2.9.0 sounds correct to me.

Comment (jasondlee) [reply .github/project.yml:3]: No, I was targeting 2.8.2, but that was before the otel/mptel upgrade, so 2.9.0 is probably fine.

File: .github/project.yml (hunk 1 above)
Comment (rhusar) [reply .github/project.yml:1]: Classic copy paste, Roberto first set this up for config, so I guess this just never got updated when porting. 

Opened https://github.com/smallrye/smallrye-opentelemetry/pull/414

**Pull request merged**